### PR TITLE
Removed need for trimIdx and fDigitsFound, resulting in less code.

### PR DESCRIPTION
**Pull request by pi1024e**
Having an extra integer variable to hold onto an i or i-1 value is inefficient, for "if (m_token.at(i) != chZero)", the code under "fDigitsFound" will always run, so it makes sense to put the fDigitsFound code under the if statement, which can return from the function entirely instead of breaking.

### How changes were validated:

- Manual Testing verified the code having identical behavior as before, with no side effects.

### CODE REVIEW

File: src/CalcManager/ExpressionCommand.cpp [hunk 1]
```diff
@@ -256,38 +256,30 @@ const wstring& COpndCommand::GetToken(wchar_t decimalSymbol)
...
 
     // Remove zeros
-    bool fDigitsFound = false;
-    int trimIdx = 0;
     for (unsigned int i = 0; i < m_token.size(); i++)
     {
         if (m_token.at(i) != chZero)
         {
-            if (m_token.at(i) == decimalSymbol)
+            if (m_token.at(i) == decimalSymbol)
```
Comment (HowardWolosky) [inline src/CalcManager/ExpressionCommand.cpp]: nit: you added a trailing space

```suggestion
            if (m_token.at(i) == decimalSymbol)
```

File: src/CalcManager/ExpressionCommand.cpp [hunk 2]
```diff
@@ -256,38 +256,30 @@ const wstring& COpndCommand::GetToken(wchar_t decimalSymbol)
...
         }
     }
-    else
-    {
-        m_token.clear();
-        m_token.append(&chZero);
-    }
+     m_token.clear();
+     m_token.append(&chZero);
+
```
Comment (HowardWolosky) [inline src/CalcManager/ExpressionCommand.cpp]: nit: you added an extra, unnecessary line

```suggestion
```

File: src/CalcManager/ExpressionCommand.cpp [hunk 3]
```diff
@@ -256,38 +256,30 @@ const wstring& COpndCommand::GetToken(wchar_t decimalSymbol)
...
+            }
+            return m_token;
         }
     }
-    else
-    {
-        m_token.clear();
-        m_token.append(&chZero);
-    }
+     m_token.clear();
```
Comment (HowardWolosky) [inline src/CalcManager/ExpressionCommand.cpp]: nit: the indentation for these lines is off after the refactor

```suggestion
    m_token.clear();
```

File: src/CalcManager/ExpressionCommand.cpp [hunk 4]
```diff
@@ -256,38 +256,30 @@ const wstring& COpndCommand::GetToken(wchar_t decimalSymbol)
...
+     m_token.append(&chZero);
```
Comment (HowardWolosky) [inline src/CalcManager/ExpressionCommand.cpp]: nit: the indentation for these lines is off after the refactor

```suggestion
    m_token.append(&chZero);
```

**Comment by pi1024e** [comment]
Made those formatting changes. Thank you!

**Comment by HowardWolosky** [comment]
Thanks @pi1024e.  This update brought in a bunch of additional changes across ten more files, unrelated to the topic of the initial PR.  Was that intentional?

**Comment by pi1024e** [comment]
That was unintentional, but this actually applies the same types of code reduction changes I made in the first revision. If you like I can resend the pull request with only the first file changed.

**Comment by pi1024e** [comment]
All formatting changes applied!

**Comment by pi1024e** [comment]
Is this OK or should I only commit some of the changed files as per the original PR? Please let me know. Thank you!

**Comment by HowardWolosky** [comment]
>Is this OK or should I only commit some of the changed files as per the original PR? Please let me know. Thank you!

Let's stick with the content of your original PR.  You're welcome to submit a separate PR for the other changes, and those will be evaluated/reviewed separately.  Thanks!

**Comment by pi1024e** [comment]
All requested changes are done! This should be good to go!

**Comment by HowardWolosky** [comment]
You still have whitespace changes in two unrelated files which you need to revert first. Thanks.

**Comment by pi1024e** [comment]
Fixed!

**Pull request merged**